cg/meta/workflow/nallo.py

```diff
@@ -0,0 +1,20 @@
+"""Module for Nallo Analysis API."""
+
+import logging
+from cg.constants import Workflow
+from cg.meta.workflow.nf_analysis import NfAnalysisAPI
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+class NalloAnalysisAPI(NfAnalysisAPI):
+    """Handles communication between NALLO processes
```

Comment:
Unify the way of writing the name of the pipeline. I would suggest Nallo but I have no real preference, as long as it is consistent
```suggestion
    """Handles communication between Nallo processes
```